frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -286,12 +286,19 @@ class Worksheet extends React.Component {
         // state can take the value of 0 or 1
         // 0 represents the command is being executed
         // 1 represents the command has already been executed
-        const cmdMsgMap: { string: string } = { rm: ['deleting', 'deleted'] };
-        let toastMsg =
-            (command in cmdMsgMap
-                ? count + (count === 1 ? ' bundle ' : ' bundles ') + cmdMsgMap[command][state]
-                : (state === 0 ? 'Executing ' : 'Executed ') + command + ' command') +
-            (state === 0 ? '...' : '!');
+        const cmdMsgMap = { rm: ['Deleting', 'deleted'] };
+        const bundleCount = count + (count === 1 ? ' bundle' : ' bundles');
+        const cmdMsg = cmdMsgMap[command] ?? ['Executing', 'Executed'];
+        let toastMsg;
+        if (command in cmdMsgMap) {
```

Comment:
Since we've substantially changed the logic here, and it's not super trivial, I think it might be good if we start writing some unit tests for this code.

Could you refactor this into a separate function `getToastMsg` (which is outside the component)? Then, we can add tests in this file (https://github.com/codalab/codalab-worksheets/blob/eabf0759dcb337cf125b4871908fa858b9434cbf/frontend/src/__tests__/Worksheet.test.js) such as the following:

```js
expect(getToastMsg("delete", 1, 1)).toEqual("Deleting 1 bundle...");
expect(getToastMsg("delete", 1, 2)).toEqual("Deleting 2 bundles...");
```